library/userHelper.ts

```diff
@@ -9,16 +9,36 @@ export async function checkUserHasSufficientSats(
   userId: string,
   msatAmount: number
 ): Promise<boolean> {
+  const inflight = await db
+    .knex("external_payment")
+    .select(
+      "external_payment.msat_amount as msatAmount",
+      "external_payment.fee_msat as feeMsat"
+    )
+    .select(
+      db.knex.raw(`SUM(external_payment.msat_amount) OVER () as "totalAmount"`)
+    )
+    .select(db.knex.raw(`SUM(external_payment.fee_msat) OVER () as "totalFee"`))
+    .where("external_payment.in_flight", "=", true)
+    .andWhere("external_payment.user_id", "=", userId)
+    .first();
```

Comment:
Maybe cleaner? (not tested)
```suggestion
  const inflight = await db
    .knex("external_payment")
    .sum("msat_amount as totalAmount")
    .sum("fee_msat as totalFee")
    .where("in_flight", "=", true)
    .andWhere("user_id", "=", userId)
    .groupBy("user_id")
    .first();
```